README.md

```
@@ -5,7 +5,9 @@ Repository for code related to small tasks / validation work for supporting soli
 |   ---     |     ---     |
 | [EBH-2633] | TSO data for MSI (including total MSI sites and unstable sites) and TMB (including CDS and count of eligible variants) since it started, for a validation project |
 | [EBH-4136] | Finds and parses out tables from WGS supplementary HTML files and aggregates into single TSVs. |
+| [DI-2086]  | Code scripts used for TSO500 validation for the S2 Flowcell
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)